XSUP-27659 DEMISTO_SDK_IGNORE_CONTENT_WARNING not effective in repo without remote / remote named origin (#3606)
demisto/demisto-sdk: README.md

      unset XSIAM_AUTH_ID
      ```

>       In case your main git branch is not named **master**, or your upstream is not named **origin**, you can set the following env vars:
>       ```bash
>       export DEMISTO_DEFAULT_BRANCH = main_barnch_title
>       export DEMISTO_DEFAULT_REMOTE = upstream_title

Reviewer: ```suggestion
      In case the primary git branch is not **master**, or the upstream is not named **origin**, set them with environment variables:
      ```bash
      export DEMISTO_DEFAULT_BRANCH = <branch name here>
      export DEMISTO_DEFAULT_REMOTE = <upstream name here>
```
